harness/determined/common/api/experiment.py

```diff
@@ -27,7 +27,9 @@ def patch_experiment(master_url: str, exp_id: int, patch_doc: Dict[str, Any]) ->
 
 
 def activate_experiment(master_url: str, exp_id: int) -> None:
```

Comment:
question: do we still need this with this change you've done https://github.com/determined-ai/determined/pull/3418/files?show-viewed-files=true&file-filters%5B%5D=#diff-46809115da97aa7d7f43e797b3b733525ec317cd5535494d6022d733acf06079R40

Reply:
I see in in create_experiment 👍🏼